### PR TITLE
contributing guidelines

### PR DESCRIPTION
**Pull request by awalterschulze**
This sounds so official, but I just thought it might be good to make sure we are on the same page.
This also hopefully makes it simple for new comers.

### CODE REVIEW

**Comment by awalterschulze** [comment]
Here are the TODOs updated to conform to the guidelines
https://github.com/awalterschulze/regex-reexamined-coq/pull/16

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+1. Find a TODO or Admitted proof and try to prove it.
+2. Read the paper and find something we haven't proved yet.
+3. Open an github issue, if you would like to prove something different that you still feels belong here.
+
+### I have an alternative proof
+
+The more the merrier.
+
+We welcome alternative proofs.  Please submit your proof with a new name, by adding a tick `'` at the end of the function name.
```
Comment (mheiber) [inline CONTRIBUTING.md]: Nice!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Reason: This is a learning repo and seeing alternatives to the same proof is helpful.  This is not just helpful in pull requests, but also to new comers to the repo and that is why it is still useful to merge them into master.
+
+### Comments
+
+ - Do we like comments? Yes we do.
```
Comment (mheiber) [inline CONTRIBUTING.md]: Do you have a link handy to a repo with good comment style? I'm still learning the conventions in the Coq community.

Comment (awalterschulze) [reply CONTRIBUTING.md]: No I don't, but if you do find one, please share.
At the moment best effort is enough.

**Pull request merged**